### ISSUE TITLE
doesn't seems to work on android and ios

### ISSUE THREAD

**Issue by fxyong** [react-community/lottie-react-native#350]
I've set-up and followed the tutorial on this link: http://airbnb.io/lottie/react-native/react-native.html#getting-started 
Using the below code... 

```
import React from 'react';
import LottieView from 'lottie-react-native';

export default class BasicExample extends React.Component {
  componentDidMount() {
    this.animation.play();
    // Or set a specific startFrame and endFrame with:
    this.animation.play(30, 120);
  }

  render() {
    return (
      <LottieView
        ref={animation => {
          this.animation = animation;
        }}
        source={require('../path/to/animation.json')}
      />
    );
  }
}
```

However, it doesn't seem to work on both platforms. It either says: this.animation.play is undefined or the app works, but nothing is showing.
Any help is much appreciated. Thanks

**Comment by emilioicai** [comment]
Which version of lottie-react-native are you using? is the problem still there if you use older versions?

**Comment by jdnichollsc** [comment]
The same from iOS with the version: `^2.5.6`

**Comment by emilioicai** [comment]
this should be now fixed on `^2.5.8`
